app/views/layout.server.view.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
I really like these changes by using Jade more efficiently and demonstrating great use of layouts. :+1: